client/pdo/client/controller/commands/eservice.py

```diff
@@ -105,4 +105,4 @@ def get_enclave_service(state, enclave_url=None) :
     if enclave_url is None :
         raise Exception('no enclave service specified')
 
-    return EnclaveServiceClient(enclave_url)
+    return EnclaveServiceClient(enclave_url)
```

Comment:
suggest you just revert this particular change

Reply:
yes, i should have , will fix